topology/INFN-T1/INFN-T1/INFN-T1CachingInfrastructure.yaml

```
@@ -0,0 +1,7 @@
+  T1_STASHCACHE_CACHE:
+    FQDN: xcache.cr.cnaf.infn.it
+    Service: XRootD cache server
+      Description: StashCache cache server 
+    AllowedVOs:
+      - ANY_PUBLIC
+    DN: DC=org, DC=terena, DC=tcs, C=IT, L=Frascati, O=Istituto Nazionale di Fisica Nucleare, CN=ds-102-11-18.cr.cnaf.infn.it
```

Hi, we use the older openssl DN formatting:

```suggestion
    DN: /DC=org/DC=terena/DC=tcs/C=IT/L=Frascati/O=Istituto Nazionale di Fisica Nucleare/CN=ds-102-11-18.cr.cnaf.infn.it
```